src/Umbraco.Web.UI.Client/src/views/components/umb-node-preview.html

```diff
@@ -3,7 +3,7 @@
         <i ng-if="icon" class="umb-node-preview__icon {{ icon }}" aria-hidden="true"></i>
         <div class="umb-node-preview__content">
 
-            <div class="umb-node-preview__name" ng-attr-title="{{alias}}">{{ name }}</div>
+            <div class="umb-node-preview__name" ng-attr-title="{{::showNodeAlias ? alias : null}}">{{ name }}</div>
```

Comment:
This is using double colon/one-time binding syntax now, but doesn't that prevent updating the value after the `userService.getCurrentUser()` promise is fulfilled and asynchronously updates `scope.showNodeAlias`?

Reply:
Good point, It did not prevent that, but for safety lets avoid it. A change is coming very soon.